welcome.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+slug: /
+---
+
+UMA is open-source infrastructure for deploying and enforcing synthetic assets on Ethereum. UMA enables developers to quickly and easily build synthetic tokens that track the price of anything! View [here](synthetic-tokens/what-are-synthetic-assets.md) to learn more about synthetic tokens!
+
+
+## What infrastructure does UMA provide?
+
+UMA enables developers to create synthetic assets via two core components:
+
+- [Priceless financial contract templates](https://docs.umaproject.org/getting-started/synthetic-tokens)
+- [Optimistic oracle service](https://docs.umaproject.org/getting-started/oracle)
+
+Together, these two components enable the creation of fast, efficient, and secure synthetic derivatives on the Ethereum blockchain.
+
+## Understanding UMA
+
+If you would like to learn more about the concepts behind UMA's priceless financial contracts and optimistic oracle, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem.
+
+## For Developers
+
+If you are a developer interested in building with UMA Protocol, view this [explainer](synthetic-tokens/what-are-synthetic-assets.md) of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens. 
+
+Additionally, the [development guide](build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA. 
+
+You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md).
+
+## For UMA Tokenholders
+
+If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
+
+## Contact 
+If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+---
+
+UMA stands for "Universal Market Access" 
+
+<!--taken from UMA/TRS Mission and Values -->
+Our mission is to make financial markets universally fair and accessible.
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
+##What is UMA##

```

Comment:
This is in the overview page. No need to be on a welcome page. The What infrastructure does uma provide section should suffice. This is intended to be a navigation site to direct people to the right place